apps/chat/src/components/Chat/Publish/PublishWizard.tsx

```diff
@@ -374,13 +374,21 @@ export function PublishModal({
                   {t('Publish to')}
                 </label>
                 <button
-                  className="input-form mx-0 flex grow items-center justify-between rounded border border-primary bg-transparent px-3 py-2 placeholder:text-secondary hover:border-accent-primary focus:border-accent-primary focus:outline-none"
+                  className="input-form mx-0 flex grow items-center rounded border border-primary bg-transparent px-3 py-2 placeholder:text-secondary hover:border-accent-primary focus:border-accent-primary focus:outline-none"
```

Comment:
most of this classes already in `button button-secondary`

Reply:
I used input-form and button to reduce the repeating styles